config/locales/en.yml

```diff
@@ -57,3 +61,22 @@ en:
       full: "%B %-d, %Y"
       youth_date_of_birth: "%B %Y"
       short_date: "%-m/%d"
+  activerecord:
+    models:
+      judge:
+        one: Judge
+        other: Judges
+    attributes:
+      judge:
+        name: Name
+  new_record: "New %{model}"
+ #create_record: "Create %{model}"
+  edit_record: "Edit %{model}"
+ #update_record: "Update %{model}"
+  destroy_record: "Delete %{model}"
+  page:
+    judge:
+      new_title: "New Judge"
+  page:
```

Comment:
YAML doesn't allow repeated keys like this- see https://stackoverflow.com/questions/47668308/duplicate-key-in-yaml-configuaration-file for an example

Reply:
For views you can also do something like
```yml
# config/locals/en.yml
judges:
  new:
    title: 'New Judge'
  edit:
    title: 'Edit Judge'
```

And in the view you can just reference it like below.
```ruby
# app/views/judges/edit.html.erb
<%= render partial: "form", locals: {title: t('.title'), judge: @judge} %>

# app/views/judges/new.html.erb
<%= render partial: "form", locals: {title: t('.title'), judge: @judge} %>
```

More info [here](https://guides.rubyonrails.org/i18n.html#lazy-lookup).